docs/source/events/examples/payment-status-change.yaml

```diff
@@ -17,5 +17,17 @@ currency_code: EUR
 # Payment method specific data (stripe intent ID in this case)
 payment_data:
   transaction_id: pi_3NOKIODvbS6ezf4w21zIR6Ft
+# Line items:
+line_items:
+- name: one-off
+  amount: 10
+  quantity: 9
```

Comment:
What does quantity: 9 mean? 9 one-off donations? If so, seems unlikely that this would happen so it's confusing.

Reply:
9 tickets to an event, 9 christmas presents,… :wink:

Reply:
The main documentation for payment_status_change events (`payment-status-change.rst`) explains that this is usually `quantity=1` … and that you only need to care about these events for advanced use cases.